### DISCUSSION TITLE
posts/2017-01-02-how-to-compare-two-groups-with-robust-bayesian-estimation-using-r-stan-and-brms/

### DISCUSSION THREAD

**Discussion by giscus** [mvuorre/mvuorre.github.io#16] [giscus]
# posts/2017-01-02-how-to-compare-two-groups-with-robust-bayesian-estimation-using-r-stan-and-brms/

2017 will be the year when social scientists finally decided to diversify their applied statistics toolbox, and stop relying 100% on null hypothesis significance testing (NHST). A very appealing alternative to NHST is Bayesian statistics, which in itself contains many approaches to statistical inference. In this post, I provide an introductory and practical tutorial to Bayesian parameter estimation in the context of comparing two independent groups’ data.

https://mvuorre.github.io/posts/2017-01-02-how-to-compare-two-groups-with-robust-bayesian-estimation-using-r-stan-and-brms/

**Comment by jackkuipers** [comment]
Small thing, but the subscripts of \mu and \sigma should just be k, not ik

**Reply by mvuorre** [reply]
Thanks so much! This is fixed in 2f8f632d089668317fdcd77143d3acdbbc999157.
